### ISSUE TITLE
Add tasks to edit individual patches

### ISSUE THREAD

**Issue by Phoenix616** [PaperMC/paperweight#22]
The original `paper` tool had [editing commands](https://github.com/PaperMC/Paper/blob/ver/1.16.5/paper#L231-L233) which simplified the cherry picking and patch creation after editing.

Would be nice to have such a feature in the new tool chain too (and maybe even a bit better working as the old one re-build all patches instead of only the edited folders and also didn't allow for editing both in parallel)

**Comment by Machine-Maker** [comment]
What else did those commands do except just cd into the directory and call `git rebase -i`?

**Comment by DenWav** [comment]
Yeah I don't recall them doing anything fancy, probably wouldn't be hard.

**Comment by Proximyst** [comment]
Could be helpful but low priority, quite frankly. Patch rebuilds are *fast* and it's doable with a normal `break` like in `git rebase -i`.

**Comment by Phoenix616** [comment]
It also automatically rebuilds the type of patches that you changed when the rebase is finished.

My usually workflow is something like that:

```
paper e s
[pick patch and edit]
paper e c
```
and it will be rebuilt and ready

**Comment by DenWav** [comment]
This may be only doable as a bash script due to the nature of interactive rebase.

**Comment by DenWav** [comment]
Unless the patch number was given as a task argument maybe, to  avoid the interactive rebase.

**Comment by electronicboy** [comment]
continue was pretty flawed last I knew, to the degree that I'm not really sad about it going
part of me does want to restore some parts of the old paper script, so I do wonder if this might be a bit of a contender there as it's not really essential, I think most of us are already in the flow of just using `git rebase -i base`, especially as the scripts did have some flaws around continuation (relied on a env variable rather than any flag files, so, couldn't edit them both together or something, iirc)

**Comment by DenWav** [comment]
If this is handled at all it will be in a script: PaperMC/Paper#6009.

I'm still of the opinion it's best to just not have this functionality at all, as the process of editing patches is simple and less likely to cause issues if done manually, but I may be outvoted on that (which is fine). In any case, if it will exist it'll be in that script.
